#3           1-JUN-1994 20:46:43.66                                  NEWMAIL
From:   MX%"vms-emacs-testers@e.kth.se"
To:     MX%"vms-emacs-testers@e.kth.se"
CC:     
Subj:   emacsclient functionality (was: Using Emacs as a MAIL/NEWS editor)

X-ListName: "A mailing list for people testing GNU Emacs on VMS."
    <vms-emacs-testers@e.kth.se>
Warnings-To: <>
Errors-To: owner-vms-emacs-testers@e.kth.se
Sender: owner-vms-emacs-testers@e.kth.se
Date: Wed, 1 Jun 94 14:53:00 EDT
From: "Roland B Roberts, 8*793-6458" <roberts@vaxms2.dnet.ge.com>
Reply-To: vms-emacs-testers@e.kth.se
X-To: "vms-emacs-testers@e.kth.se"
To: vms-emacs-testers@e.kth.se
Subject: emacsclient functionality (was: Using Emacs as a MAIL/NEWS editor)

Having posted my code, naturally I found a problem.  I changed two lines
to deal with search lists a little better.  Here's a patch

*** emacsclient.com-old Wed Jun  1 14:40:51 1994
--- emacsclient.com     Wed Jun  1 14:40:41 1994
***************
*** 44,50 ****
  $   file = f$edit(f$extract(0,x,args),"TRIM")
  $   args = f$extract(x+1,l-x-1,args)
  $   if file .eqs. "" .and. args .nes. "" then goto build_list
! $   temp = f$parse(file,,,,"SYNTAX_ONLY")
  $   if temp .nes. ""
  $   then
  $     if f$locate(";",file) .eq. f$length(file)
--- 44,51 ----
  $   file = f$edit(f$extract(0,x,args),"TRIM")
  $   args = f$extract(x+1,l-x-1,args)
  $   if file .eqs. "" .and. args .nes. "" then goto build_list
! $   temp = f$search(file)
! $   if temp .eqs. "" then temp = f$parse(file,,,,"SYNTAX_ONLY")
  $   if temp .nes. ""
  $   then
  $     if f$locate(";",file) .eq. f$length(file)

